components/app-core/backend/cnsi.go

```diff
@@ -46,7 +46,15 @@ func (p *portalProxy) RegisterEndpoint(c echo.Context, fetchInfo interfaces.Info
 		skipSSLValidation = false
 	}
 
-	newCNSI, err := p.DoRegisterEndpoint(cnsiName, apiEndpoint, skipSSLValidation, fetchInfo)
+	cfclientid :=  c.FormValue("cf_client_id")
```

Comment:
Can we make these vars and the form values generically named - ie. loose the cf prefix. We will support other endpoint types over time.